test/browser_test.rb

```diff
@@ -65,6 +65,15 @@ class BrowserTest < Minitest::Test
     assert_equal "4", @browser.version
   end
 
+  test "detects chrome browser" do
+    @browser.ua = $ua["CHROME"]
```

Comment:
not sure what you mean: other tests also use predefined user agent keys from ua.yml, e.g. @browser.ua = $ua["ANDROID"], "CHROME" is one of them

Do you prefer to do @browser.ua = "Chrome" instead?